src/github.com/getlantern/enproxy/conn_test.go

```diff
@@ -91,26 +100,23 @@ func doTestTLS(buffered bool, t *testing.T) {
 
 	doRequests(tlsConn, t)
 
-	if bytesReceived != 555 {
-		t.Errorf("Bytes received of %d did not match expected %d", bytesReceived, 555)
-	}
-	if bytesSent != 4968 {
-		t.Errorf("Bytes sent of %d did not match expected %d", bytesSent, 4968)
-	}
+	assert.Equal(t, 550, bytesReceived, "Wrong number of bytes received")
+	assert.Equal(t, 1580, bytesSent, "Wrong number of bytes sent")
```

Comment:
Good point.  The amount of data depends on things that may be beyond my control.  Since we know that counting bytes works from the plain text test, I'll just take out those conditions.